Exercise_5.java

```diff
@@ -1,36 +1,50 @@
+package precourse2;
+
+import java.util.*; 
+
 class IterativeQuickSort { 
-    void swap(int arr[], int i, int j) 
+    static int partition(int arr[], int low, int high) 

```

Comment:
Good Job, You have mistakenly swapped implementation of iterative and recuusive QuickSort, I think. It's fine for now.